### PR TITLE
docs: update content of chapters 1 and 2

### LINKED ISSUE #316: Content changes and Inline code block of JS Course -- Chapter 1 and 2.
Content changes and Inline code block of JS Course -- Chapter 1 and 2.

**Steps To Reproduce**
1. Go to https://test-app.nullcast.dev/curriculum
2. Click on the Hyperlink "School of Ducks" in the header.
3. Scroll down to the Curriculum 
4. Click on "Learn More" button of Javascript, where User gets redirected to the Javascript Course page.
5. Click on the "Start" button, where the users gets redirected to the Javascript chapters page.
6. Click on the Bottom Hamburger icon where the Nav bar containing the menu of all the chapters listed.
7. User while learrning through the chapters 1 and 2, could see some issues in contents and some technical terms which need to be given the Inline code block (single back tick), which are marked inside boxes.

**Expected behavior**
Content Changes and the Inline code block to be corrected in the JS chapters 1 and 2.

**Screenshots attached**

**Desktop (please complete the following information):** ThinkPad
 - OS: Linux
 - Browser: Google Chrome 
 - Version: Version 95.0.4638.54 (Official Build) (64-bit)

**Additional context**
Tested in Testsite
2Hello-World-Curriculum4
2Hello-World-Curriculum3
2Hello-World-Curriculum2
2Hello-World-Curriculum1
1What is JavaScript

### PR DESCRIPTION
**Pull request by reginaalyssa**
# Pull Request Template

## Description

* Updated content and spacing for `courses/javascript/introduction.md`
* Updated content for `courses/javascript/hello-world.md`
* Added inline code blocks
* Fixes spacing issues
* Updated mostly grammar and capitalizations (html to HTML, javascript to JavaScript)

Introduction changes with spacing:
image

Fixes #316 

## Type of change

- [x] This change requires a documentation update

## How Has This Been Tested?

Run the server locally and visit the following pages to verify the updated content:
http://localhost:3000/curriculum/javascript
http://localhost:3000/curriculum/javascript/hello-world

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [N/A] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [N/A] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: courses/javascript/hello-world.md [hunk 1]
```diff
@@ -14,15 +14,15 @@ testCase: [
 		]
 ---
 
-In this section we are going to see how you can execute **javascript** code on browser.
+In this section we are going to see how you can execute **JavaScript** code on browser.
 
 <br />
 
 **There are Three main ways to execute JavaScript on browser.**
```
[the remark this answers is not in the document]
Comment (thealpha93) [reply courses/javascript/hello-world.md:21]: Nice find @reginaalyssa .
There's no missing part of the document. Can you update it to "two"?

File: courses/javascript/hello-world.md [hunk 2]
```diff
@@ -35,12 +35,13 @@ console.log("hi");
...
 </body>
 ```
+<br/>
 
-But the HTML5 introduce 2 attribute that help us to define when we want to execute our code.
+But HTML5 introduces 2 attributes that help us to define when we want to execute our code.
+<br/><br/>
+#### 1. [`defer`](https://www.w3schools.com/tags/att_script_defer.asp)
 
-#### 1. [defer](https://www.w3schools.com/tags/att_script_defer.asp)
```
[the remark this answers is not in the document]
Comment (thealpha93) [reply courses/javascript/hello-world.md:41]: Good suggestion. Can you use `i` and `ii` instead of 1 and 2?

**Comment by thealpha93** [comment]
Also @reginaalyssa Can you send the PR to `development` branch? The `development` branch has the latest code in it. Thanks

File: courses/javascript/hello-world.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (reginaalyssa) [reply courses/javascript/hello-world.md:21]: Got this, thank you!

File: courses/javascript/hello-world.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (reginaalyssa) [reply courses/javascript/hello-world.md:41]: Will do! Thank you so much. The files have been updated accordingly!

**Comment by reginaalyssa** [comment]
Hi @thealpha93 this PR is ready for review. Please let me know if there are any additional changes needed. 😄

**Comment by thealpha93** [comment]
@reginaalyssa thanks. It’s been merged.
Would you like join our discord community and collaborate more closely with us??

**Pull request merged**